Source/common/SNTBlockMessage.mm

```diff
@@ -111,6 +111,15 @@ + (NSAttributedString *)attributedBlockMessageForDeviceEvent:(SNTDeviceEvent *)e
                            withFallback:defaultBannedMessage];
 }
 
++ (NSAttributedString *)attributedBlockMessageForNetworkMountEventWithCustomMessage:
+    (NSString *)customMsg {
+  NSString *defaultBannedMessage =
+      NSLocalizedString(@"The following network share has been blocked from mounting",
+                        @"The default message to show the user when a network mount is blocked");
```

Comment:
This and a [few other strings](https://github.com/northpolesec/santa/actions/runs/20542089785/job/59007791429?pr=708) need to be localized. I've opened another PR to fix the script so that it causes the check to fail appropriately, which also adds the "Clean Sync" string.